src/config.ts

```diff
@@ -299,3 +334,12 @@ export function getStatusProviderFromConfig(): BaseStatusProvider {
     }
   }
 }
+
+export function getChangelogPolicyFromConfig(): ChangelogPolicy {
+  const config = getConfiguration() || {};
```

Comment:
Why do we need the `|| {}` here but not above?

Reply:
Just copy pasted. I can do a go over for all usages.